app/widgets/left-nav.js

```diff
@@ -0,0 +1,21 @@
+import Vue from "vue";
+
+import { NAV_LINKS } from "../data";
+
+var LeftNav = Vue.component("left-nav", {
+  template: "#tpl-widgets-left-nav",
+  data() {
+    return {
+      links: NAV_LINKS
+    };
+  },
+  methods: {
```

Comment:
looks like it did need the `goto` because pages are not scrolling correctly. Need to fix that with the Vue Router.